Eventy/src/app/shared/solution-category-input-dialog/solution-category-input-dialog.component.html

```diff
@@ -0,0 +1,13 @@
+<h2 mat-dialog-title>{{message}}</h2>
```

Comment:
Should this dialog be in a shared module? Will it be used by more than one other module?

Reply:
A well placed comment! I believe it will be able to be used by the service and product modules for the creation of each of them. A PUP will be able to create their own categories, which will later need to be approved by an admin.